src/familybot/plugins/steam_admin.py

```
@@ -446,81 +373,86 @@ async def force_deals_unlimited_command(self, ctx: PrefixedContext):
+                        if not game_data.get("is_family_shared", False):
                             continue
-                        game_data = game_info_json.get(str(app_id), {}).get("data")
-                        if not game_data:
+                        game_name = game_data.get("name", f"Unknown Game ({app_id})")
+                        # Handle both cached data (price_data) and fresh API data (price_overview)
+                        price_overview = game_data.get(
+                            "price_overview"
+                        ) or game_data.get("price_data")
+                        if not price_overview:
+                            logger.debug(
+                                f"Force deals unlimited: No price data found for {app_id} ({game_name})"
```

Ruff (0.15.1)

[warning] Logging statement uses f-string

(G004)